Makefile

```diff
@@ -87,6 +87,7 @@ endif
 
 .PHONY: bootstrap-database
 bootstrap-database:
+	cp fixtures/jobs.yml vendor/akeneo/pim-community-dev/src/Akeneo/Platform/Bundle/InstallerBundle/Resources/fixtures/minimal/jobs.yml
 	APP_ENV=prod $(MAKE) database O="--catalog vendor/akeneo/pim-community-dev/src/Akeneo/Platform/Bundle/InstallerBundle/Resources/fixtures/minimal"
```

Comment:
Zomg that string. It's duplicated directly below as well... Is there an easy way to parameterize that in a Makefile?

Reply:
Fixed!